attributes/default.rb

```diff
@@ -24,41 +24,63 @@
 
 default['telegraf']['download_urls'] = {
   'debian' => 'https://dl.influxdata.com/telegraf/releases',
-  'rhel' => 'https://dl.influxdata.com/telegraf/releases'
+  'rhel' => 'https://dl.influxdata.com/telegraf/releases',
+  'windows' => 'https://dl.influxdata.com/telegraf/releases',
 }
 
 # platform_family keyed download sha256 checksums
 default['telegraf']['shasums'] = {
   'debian' => '',
-  'rhel' => ''
+  'rhel' => '',
+  'windows' => '',
 }
 
-default['telegraf']['config_file_path'] = '/etc/telegraf/telegraf.conf'
+# rubocop:disable LineLength
+default['telegraf']['config_file_path'] = platform_family?('windows') ? "#{ENV['ProgramW6432']}\\telegraf\\telegraf.conf" : '/etc/telegraf/telegraf.conf'
+# rubocop:enable LineLength
 
 default['telegraf']['config'] = {
   'tags' => {},
   'agent' => {
     'interval' => '10s',
     'round_interval' => true,
     'flush_interval' => '10s',
-    'flush_jitter' => '5s'
-  }
+    'flush_jitter' => '5s',
+  },
```

Comment:
despite my affinity for them, we are avoiding commas as the last item in a hash.

`C: Avoid comma after the last item of a hash.`

Reply:
Well fine by me. The latest cookstyle has them enabled and since you didn't disable the rule in the rubocop.yml. I just made sure all foodcritic/foodstyle errors were gone incase you want to use a CI pipeline.

I could disable the rule in the rubocop.yml if you like.

Reply:
Okay, let me check again.  I saw them by running rubocop, but maybe my rubocop is out of date (again... :)

Reply:
OK this looks good after getting everything updated... I'll need to add some linting notes to the README I think.

Reply:
Just let me know if you want this linted with cookstyle or rubocop. I tend to follow cookstyle for chef things.

Reply:
I'm fine with cookstyle, we've started to use it at work as well.  I'll make sure the README gets updated